deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -86,14 +89,17 @@
   private static final ClassName CLASS_COLLECTIONS = ClassName.get(Collections.class);
   private static final Class<DeepLink> DEEP_LINK_CLASS = DeepLink.class;
   private static final Class<DeepLinkSpec> DEEP_LINK_SPEC_CLASS = DeepLinkSpec.class;
+  private static final String DOC_OUTPUT_PROPERTY_NAME = "deepLinkDoc.output";
```

Comment:
Sample output:

CustomPrefixesActivity: https://airbnb.com/users
CustomPrefixesActivity: https://airbnb.com/user/{id}
CustomPrefixesActivity: library://dld/library_deeplink
CustomPrefixesActivity: library://dld/library_deeplink/{lib_id}
MainActivity#intentForTaskStackBuilderMethods: http://example.com/deepLink/{id}/{name}/{place}
MainActivity#intentForParamDeepLinkMethod: dld://host/somePath/{arbitraryNumber}
MainActivity: dld://classDeepLink
MainActivity: http://example.com/foo{arg}
MainActivity: dld://example.com/deepLink
MainActivity#intentForComplexMethod: dld://host/somePathOne/{arbitraryNumber}/otherPath

The location of the file is specified in the gradle of the sample project, currently at ${buildDir}/doc/deeplinks.md